Cargo.toml

```diff
@@ -1,10 +1,21 @@
 [package]
 edition = "2021"
 name = "datafusion-objectstore-s3"
+description = "S3 as an ObjectStore for Datafusion"
 version = "0.1.0"
+homepage = "https://github.com/datafusion-contrib/datafusion-objectstore-s3"
+repository = "https://github.com/datafusion-contrib/datafusion-objectstore-s3"
+readme = "../README.md"
+authors = ["Matthew Turner <matthew.m.turner@outlook.com>", "Michael Seddon <>"]
```

Comment:
@seddonm1 can you let me know your email so i can add?

Reply:
seddonm1@gmail.com